store/modules/actions/authCore.js

```diff
@@ -130,9 +132,24 @@ export async function prepareAuthCoreCosmosTxSigner({ state }) {
   if (!state.cosmosProvider) throw new Error('COSMOS_WALLET_NOT_INITED');
   const { cosmosProvider } = state;
   return {
-    signAmino: async (_, data) => {
-      const { signatures, ...signed } = await cosmosProvider.sign(data);
-      return { signed, signature: signatures[0] };
```

Comment:
Can we actually keep `signAmino` here without using it, and let cosmjs stargate use `signDirect` ?